src/main/java/salesforce/utils/StringDateConverter.java

```diff
@@ -0,0 +1,116 @@
+package salesforce.utils;
```

Comment:
You should add the copyright, to match the code convention.

Reply:
ok